notebooks/240-efficientdet-object-detection/240-efficientdet-object-detection.ipynb

```diff
@@ -0,0 +1,624 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "js4jU4zhchQ_"
+      },
+      "source": [
+        "# EfficientDet with OpenVINO™\n",
+        "Research Paper : [EfficientDet: Scalable and Efficient Object Detection](https://openaccess.thecvf.com/content_CVPR_2020/papers/Tan_EfficientDet_Scalable_and_Efficient_Object_Detection_CVPR_2020_paper.pdf)\\\n",
+        "Official GitHub : [EfficientDet](https://github.com/google/automl/tree/master/efficientdet)\n",
+        "\n",
+        "EfficientDets are a family of object detection models, which achieve state-of-the-art 55.1mAP on COCO test-dev, yet being 4x - 9x smaller and using 13x - 42x fewer FLOPs than previous detectors. EfficientDet models also run 2x - 4x faster on GPU, and 5x - 11x faster on CPU than other detectors.\n",
+        "\n",
+        "![Model_structure](https://raw.githubusercontent.com/google/automl/master/efficientdet/g3doc/network.png)\n",
+        "![Params](https://user-images.githubusercontent.com/71766106/226909727-2d321609-9c2c-4e25-a43a-6555778408d6.png)\n",
+        "\n",
+        "Achieving similar accuracy with fewer parameters is the benefit of efficientdet as it makes the training and predicting process efficient and fast hence scalable.\n",
+        "\n",
+        "\n",
+        "## The tutorial consists of the following steps:\n",
+        "\n",
+        "* Downloading and exporting the efficientDet-d0 from TensorFlow-HUB.\n",
+        "  * Validating inference of tf-model.\n",
+        "* Converting tf-model to OpenVINO-IR (Intermediate representation) file.\n",
+        "  * Validating inference of OpenVINO-model.\n",
+        "  * Evaluating performance by AP (average Precision).\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "NOYWoDWfEyRD"
+      },
+      "source": [
+        "### Prerequisites"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "xAKESkEz16iP"
+      },
+      "outputs": [],
+      "source": [
+        "# These dependencies will be added to the requirements.txt once approved.\n",
+        "!pip install openvino-dev[tensorflow2] tensorflow-hub pycocotools"
```

Comment:
Thanks for contributing to the EfficientDet-optimization-notebook. I recommend moving the software requirements mentioned in the notebook to a separate file, such as requirements.txt , instead of having them in the notebook code itself. This will improve the clarity and maintainability of the notebook code.

Reply:
Correct me if I am wrong, but I think we are not allowed to have any other files except notebook and readme.